public/app/plugins/datasource/grafana-azure-monitor-datasource/resourcePicker/resourcePickerData.ts

```diff
@@ -68,14 +68,15 @@ export default class ResourcePickerData extends DataSourceWithBackend<AzureMonit
 
     return resources.map((subscription) => ({
       name: subscription.subscriptionName,
-      id: subscription.subscriptionId,
+      id: `/subscriptions/${subscription.subscriptionId}`,
```

Comment:
this is confusing, some times this code is using `id` as the resource ID (like it was before this change), and some others it's using an URI. Can we change the name of this to `uri` if we are going to use it always as an URI?

Reply:
I'll change the name to uri I think? The problem is with resource groups and resources we've been saving the id as a URI, but not for subscriptions. I could attempt to solve this another way by just storing the id and crafting the uri wherever we need it, but it felt like there might be risk of breaking people's dashboards in doing so...but now I'm thinking about it and I'm not entirely sure that's really a risk. 🤔